docs/README_sp.md

```diff
@@ -0,0 +1,195 @@
+donde `$DOMAIN` es el dominio personalizado que configuró `env.mk`.
+
+## Actualizar la Aplicación
+
+Ejecutar `make build-all` hará el build y desplegará la aplicación, incluidos los cambios realizados en la infraestructura. Tenga en cuenta que eliminar recursos de `infrastructure-tpl.yaml`
+no hará que se eliminen. Debe ejecutar `make delete` antes de eliminar el recurso (luego volver a implementar con `make build-all` después de eliminarlo), o eliminar manualmente el recurso con `kubectl delete`.
+
+```shell
+# construye e implementa infraestructura de back-end, frontend e KCC
+make build-all
+
+# construye e implementa solo el servicio de back-end Go
+make build-backend
+
+# construye y despliega solo la aplicación web angular frontend
+make build-webui
+```
+
+## Limpieza
+
+La ejecución de `make delete` eliminará los recursos del conector de configuración de su clúster, lo que hará que el conector de configuración elimine los recursos de GCP asociados. Sin embargo, debe eliminar manualmente su servicio Cloud Run y ​​GKE Cluster.
+
+[APIs & Services -> Credentials]: https://console.cloud.google.com/apis/credentials
+[Cloud Build]: https://cloud.google.com/cloud-build/docs
+[Config Connector]: https://cloud.google.com/config-connector/docs
+[Cloud DNS Managed Zone]: https://cloud.google.com/dns/zones
+[update name server records]: https://cloud.google.com/dns/docs/migrating#update_your_registrars_name_server_records
+[domain ownership verification]: https://cloud.google.com/storage/docs/domain-name-verification#verification
+[additional verified owner]: https://cloud.google.com/storage/docs/domain-name-verification?_ga=2.256052552.-234301672.1582050261#additional_verified_owners
+[Enable Identity Platform]: https://console.cloud.google.com/marketplace/details/google-cloud-platform/customer-identity
+[Identity Platform -> Providers]: https://console.cloud.google.com/customer-identity/providers
+[Identity Platform quickstart guide]: https://cloud.google.com/identity-platform/docs/quickstart-email-password#sign_the_user_in
+[Identity Platform page in the GCP console]: https://console.cloud.google.com/marketplace/details/google-cloud-platform/customer-identity
+[OAuth consent screen]: https://console.cloud.google.com/apis/credentials/consent
+[Identity Platform -> Settings]: https://console.cloud.google.com/customer-identity/settings
+[Setting up OAuth 2.0 guide]: https://support.google.com/cloud/answer/6158849?hl=en
+[set up gcloud]: https://cloud.google.com/sdk/docs
+[`makefile`]: makefile
+[Owner permission]: https://console.cloud.google.com/iam-admin/roles/details/roles%3Cowner
+[architecture.md]: ./docs/architecture.md
+[cloud-tutorial.dev]: https://cloud-tutorial.dev/
+[bootstrap.sh]: scripts/bootstrap.sh
+[firebase-config-setup.sh]: scripts/firebase-config-setup.sh
+[domain-setup.sh]: scripts/domain-setup.sh
+[firestore rules page screenshot]: docs/img/firestore_rules_page.png
+[`firestore/firestore.rules`]: firestore/firestore.rules
```

Comment:
Relative links are resolved against the path of the page they're on (try hovering over the links [here](https://github.com/GoogleCloudPlatform/cloud-run-anthos-reference-web-app/blob/4731cf629c83b41bd311b15acf25ed24d9ed6134/docs/README_sp.md) and seeing the URL in the bottom left corner of your browser). 

Since this page is in the `docs/` folder, these links need to be set relative to it.

```suggestion
[`makefile`]: ../makefile
[Owner permission]: https://console.cloud.google.com/iam-admin/roles/details/roles%3Cowner
[architecture.md]: architecture_sp.md
[cloud-tutorial.dev]: https://cloud-tutorial.dev/
[bootstrap.sh]: ../scripts/bootstrap.sh
[firebase-config-setup.sh]: ../scripts/firebase-config-setup.sh
[domain-setup.sh]: ../scripts/domain-setup.sh
[firestore rules page screenshot]: img/firestore_rules_page.png
[`firestore/firestore.rules`]: ../firestore/firestore.rules
```